coordinator.go

```diff
@@ -135,6 +140,15 @@ func newCoordinator(schemas []*tableSchema, client *spanner.Client) *coordinator
 		}
 	}
 
+	// Mark tables that has at least one global index.
+	for _, idx := range indexes {
+		// A global index isn't interleaved in any table.
+		if idx.parentTableName == "" {
+			table := tableMap[idx.baseTableName]
```

Comment:
It's possible that `tableMap` doesn't contain the table for `idx.baseTableName` because some tables could be excluded by the option `--exclude`.

So it might be better to check the existence like this:

```go
table, ok := tableMap[idx.baseTableName]
if !ok {
  continue
}
```

Reply:
Thanks for the catch, surely it can happen. I've tweaked this part like below:

```go
for _, idx := range indexes {
    // A global index isn't interleaved in any table.
    if idx.parentTableName == "" {
        if table, ok := tableMap[idx.baseTableName]; ok {
            table.hasGlobalIndex = true
        }
    }
}
```